libraries/Microsoft.Bot.Builder.Azure/AppInsightsLoggerMiddleware.cs

```diff
@@ -0,0 +1,137 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.Bot.Schema;
+
+[assembly: InternalsVisibleTo("Microsoft.Bot.Builder.Azure.Tests")]
+
```

Comment:
Personally I would remove this and just make the internal class public. 

The complexity brought by this attribute is deceiving as the Assembly Signing bits get involved and weird ordering dependencies start to happen. The root issue is that our assemblies are all signed, and that interaction with this attribute seems to cause more problems than we expected.

Reply:
Done